### DISCUSSION TITLE
hyprland-nvidia-git not working ( 12/02  )

### DISCUSSION THREAD

**Discussion by psilvaricardo** [prasanthrangan/hyprdots#602] [General]
Hi Everyone, 

Does any of you are experiencing this issue ? Any ideas of when can be fixed ?

![image](https://github.com/prasanthrangan/hyprdots/assets/125488497/6a4775c2-5478-4361-bf2c-905f0a79222d)

**Comment by OctavianOG** [comment]
Greetings @psilvaricardo , `hyprland-nvidia-git` - is deprecated and wont be no longer maintained (The devs said so in official discord), instead install `hyprland-git` - after installation process, it will detect `hyprland-nvidia-git` and suggest you to remove it, agree with it and the problem will go away.

**Comment by psilvaricardo** [comment]
Hi @OctavianOG 

That did the trick, thanks a lot.
